galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/routes/ResultNamesRoute.java

```diff
@@ -32,18 +32,18 @@
 public class ResultNamesRoute extends RunsRoute {
 
 	public ResultNamesRoute(ResponseBuilder responseBuilder, IFramework framework) {
-		/* Regex to match endpoints:
-		*  -> /ras/runs
-		*  -> /ras/runs/
-		*  -> /ras/runs?{querystring}
+		/* Regex to match endpoints: 
+		*  -> /ras/resultnames
+		*  -> /ras/resultnames?
 		*/
 		super(responseBuilder, "\\/resultnames?", framework);
```

Comment:
Not a change in your PR but this pattern matches `/ras/resultname` since the s is optional